app/experimenter/experiments/constants/nimbus.py

```diff
@@ -101,50 +101,25 @@ class NimbusTargetingConfig:
     ),
 )
 
-TARGETING_ALL_ENGLISH = NimbusTargetingConfig(
-    name="All English users",
-    slug="all_english",
-    description="All users in en-* locales.",
-    targeting="localeLanguageCode == 'en'",
-    desktop_telemetry="STARTS_WITH(environment.settings.locale, 'en')",
-    application_choice_names=(Application.DESKTOP.name,),
-)
-
-TARGETING_US_ONLY = NimbusTargetingConfig(
```

Comment:
I think this is covered by #6078, but what's the experiment owner impact of deleting these?
Previous experiments that are draft/live/completed will still show their targeting configs at the time they were created, along with some deprecation notices?

Reply:
@pdehaan Yeah that's what https://github.com/mozilla/experimenter/pull/6078 was for, so now it'll show `Deprecated` next to the targeting config for past experiments and treat drafts with deprecated targeting configs as invalid so they're forced to go back and select a valid one 👍